pkg/cmd/config/command.go

```diff
@@ -73,6 +78,27 @@ func NewCommand() *cli.Command {
 					return nil
 				},
 			},
+			{
+				Name:  "list-services",
+				Usage: "lists Kubernetes services in all namespaces",
+				Flags: []cli.Flag{
+					&cli.StringFlag{Name: "kubeconfig", Usage: "absolute path to the kubeconfig file", Required: false},
+				},
+				Action: func(c *cli.Context) error {
+					kubeconfigPath := c.String("kubeconfig")
+					config, err := clientcmd.BuildConfigFromFlags("", kubeconfigPath)
+					if err != nil {
+						return errors.Wrap(err, "building kubeconfig")
+					}
+
+					clientset, err := kubernetes.NewForConfig(config)
+					if err != nil {
+						return errors.Wrap(err, "creating clientset")
+					}
```

Comment:
We might make this into a function also. 

I think there is something [similar here](https://github.com/Cloudzero/cloudzero-agent-validator/blob/develop/pkg/diagnostic/k8s/check.go#L82-L89)

Reply:
Awesome thanks, just updated.